client/cmd/install.go

```diff
@@ -350,8 +359,11 @@ func configureZshrc(homedir, binaryPath string, skipConfigModification bool) err
 	if zshIsConfigured {
 		return nil
 	}
+
 	// Add to zshrc
-	return addToShellConfig(getZshRcPath(homedir), getZshConfigFragment(homedir), skipConfigModification)
+	return addToShellConfig(getZshRcPath(homedir),
```

Comment:
I totally understand the preference for wrapping lines, however, keeping git history clean and meaningful can be really helpful for tracking actual code changes versus formatting updates.

Would you mind reverting the line wrapping in this PR in case they don't change the code? I think it would help maintain consistency and make future diffs easier to follow. Thanks!

Reply:
But it is lint commit and does not change any logic.

Reply:
Ah, my mistake, I read the code but didn’t check the title, sorry about that. That said, this change still affects git history, making it harder to follow without adding much value.  

If the goal is to improve formatting, would it be possible to standardize and automate it? I see there's a `make fmt` command, which seems like a convenient way to keep formatting the same for everyone. Otherwise, if everyone will adjust formatting based on personal preference, it will lead only to inconsistencies, not improvements.

Reply:
I don't mind reverting it.